elixir/test.exs

```diff
@@ -0,0 +1,2 @@
+# elixir test.exs
```

Comment:
Could you add a `#!/usr/bin/env elixir` and `chmod +x` the file? Otherwise we'd need a Makefile (or something equivalent) to make it easy to compile/run.

Reply:
I'll add a `Makefile` because `#` is the comment character and the Elixir formatter will change `#!/usr/bin/env` to `# !/usr/bin/env`.

Reply:
I'd wager that could be a bug in the formatter. It should ignore the first line if it begins with `#!/...`.